lib/pbench/server/db/models/tracker.py

```diff
@@ -432,24 +432,23 @@ def create(**kwargs):
     def attach(path=None, controller=None, name=None, state=None):
         """
         attach Attempt to fetch dataset for the controller and dataset name,
-        or using a specified file path (see __init__ and _render_path).
+        or using a specified file path (see _render_path and the path_init
+        event listener for details).
 
-        If the `state` kwarg is specified, the dataset will be created in that
-        state or, if it already exists, `attach` will attempt to advance the
-        dataset to that state.
+        If state is specified, attach will attempt to advance the dataset to
+        that state.
 
         Args:
-            kwargs (dict):
-                "path": A tarball file path from which the controller (host)
-                    name, the tarball dataset name (basename minus extension)
-                    or both will be derived.
-                "controller": The controller name (hostname) of the dataset;
-                    this is retained if specified, or will be constructed
-                    from "path" if not present.
-                "name": The dataset name (file path basename minus ".tar.xz");
-                    this is retained if specified, or will be constructed from
-                    "path" if not present.
-                "state": The desired state to advance the dataset.
+            "path": A tarball file path from which the controller (host)
+                name, the tarball dataset name (basename minus extension)
```

Comment:
Got another comma handy?

Reply:
Whew; found one under the cushion on the couch.